src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityItemBus.java

```diff
@@ -170,13 +189,80 @@ private ModularUI.Builder createUITemplate(EntityPlayer player, int rowSize, int
         return builder.bindPlayerInventory(player.inventory, GuiTextures.SLOT, 7 + xOffset, 18 + 18 * rowSize + 12);
     }
 
+    private static void collapseInventorySlotContents(IItemHandlerModifiable inventory) {
+
+        // Gather a snapshot of the provided inventory
+        Map<ItemStackKey, Integer> inventoryContents = GTHashMaps.fromItemHandler(inventory);
+
+        List<ItemStack> inventoryItemContents = new ArrayList<>();
+
+        // Populate the list of item stacks in the inventory with apportioned item stacks, for easy replacement
+        for(ItemStackKey key : inventoryContents.keySet()) {
+            ItemStack stack = key.getItemStack();
+            stack.setCount(inventoryContents.get(key));
+            inventoryItemContents.addAll(InventoryUtils.apportionStack(stack, stack.getMaxStackSize()));
+        }
+
+        for(int i = 0; i < inventory.getSlots(); i++) {
+            ItemStack stackToMove;
+            // Ensure that we are not exceeding the List size when attempting to populate items
+            if(i >= inventoryItemContents.size()) {
+                stackToMove = ItemStack.EMPTY;
+            }
+            else {
+                stackToMove = inventoryItemContents.get(i);
+            }
+
+            // Populate the slots
+            inventory.setStackInSlot(i, stackToMove);
+        }
+
+    }
+
+    @Override
+    public boolean onScrewdriverClick(EntityPlayer playerIn, EnumHand hand, EnumFacing facing, CuboidRayTraceResult hitResult) {
+
+        // Check if the player is sneaking to not interfere with machine grid access
+        if (playerIn.isSneaking()){
```

Comment:
Instead of requiring sneaking, covers should be preferred if present, and then call the MTE's method if a cover was not present on that side. This requirement for sneaking should no longer be necessary.

Reply:
Covers will already be preferred if they are present, due to `MetaTileEntity#onToolClick` checking for covers first